docs/site/Project-layout-reference.md

```diff
@@ -0,0 +1,124 @@
+---
+lang: en
+title: 'Project Layout Reference'
+keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Project-layout-reference.html
+---
+
+LoopBack 4 project files and directories are in the _application root
+directory_. Within this directory the standard LoopBack 4 project structure has
+these sub-directories:
+
+- `src` - Node application scripts and configuration files.
+- `public` - Client JavaScript, HTML, and CSS files.
```

Comment:
Client side assets (JavaScript, HTML, and CSS) for the default home page.

Reply:
Maybe we should mention `dist` - which contains transpiled JavaScript code generated by `npm run build`.

Reply:
I added `dist` folder under `Top level app directory`, PTAL.